ai_agents/devable_research_agent/lib/tavily_collector.py

```
@@ -0,0 +1,279 @@
+    try:
+        response = client.search(
+            query=query,
+            search_depth=search_depth,
+            topic=topic,
+            start_date=from_date,
+            end_date=to_date,
+            max_results=max_results,
+            include_domains=include_domains,
+            timeout=60,
+        )
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)